src/pages/app/pages/Home.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { ROUTES } from '@/app/App';
+import { useAppState } from '@/app/lib/store';
+
+export const Component = () => {
+  const state = useAppState();
+  return (
+    <div className="mx-auto max-w-2xl space-y-4 p-6">
+      <div className="flex items-center justify-between">
+        <h1 className="text-2xl">Home</h1>
+        <nav>
+          <ul>
+            <li>
+              <Link
```

Comment:
this puzzles me a bit, can't we show the postinstall page directly?
why i need to click an additional link?
or is this temporary ?

Reply:
This is temporary. Will remove, and keep it as simple as possible in this PR.

Reply:
We'll have multiple screens in this `/app/index.html`, post-install being one of them. Rest unknown at the moment, but we'll have more!